pkg/elle/core/core.go

```diff
@@ -11,6 +11,18 @@ const (
 	Realtime Rel = "realtime"
 )
 
+type DependType int
+
+const (
+	RealtimeDepend DependType = iota
```

Comment:
Do not use iota and int type, it's hard to read.

Reply:
Emm what is better here?

Reply:
Perhaps the string literal is better